backend/uclapi/roombookings/management/commands/trigger_webhooks.py

```diff
@@ -155,12 +152,23 @@ def webhook_filter(booking):
         for webhook in webhooks_to_enact:
             if webhook["payload"]["content"] != {}:
                 webhook_in_db = webhook["webhook_in_db"]
+
+                webhook_success = False
+                status_code = -1
+                if "response" in webhook:
+                    status_code = webhook["response"].result().status_code
+                    if status_code not in range(400, 600):
+                        webhook_in_db.last_success = timezone.now()
+                        webhook_success = True
+
                 webhook_in_db.last_fired = timezone.now()
                 webhook_in_db.save()
 
                 new_webhook_history_entry = WebhookTriggerHistory(
                     webhook=webhook_in_db,
-                    payload=webhook["payload"]
+                    payload=webhook["payload"],
+                    success=webhook_success,
```

Comment:
I don't feel there is any need to store a boolean for `success` as this is derivable from `status_code`